website/src/contribute/release-guide.md

```diff
@@ -1248,6 +1256,14 @@ please follow [the guide](https://help.github.com/articles/creating-a-personal-a
    delete the `.asc`, `.sha512`;
 3. Upload the new release `twine upload *` from the directory with the `.zip` and `.whl` files;
 
+[Installing twine](https://packaging.python.org/tutorials/packaging-projects/):
+```
+virtualenv env
+. ./env/bin/activate
```

Comment:
For brevity, could this just say: "use `pip install twine` to install twine locally or in a virtualenv of your choice"?
Otherwise, I'd also write down to deactivate and clean up env/ when done.